### PR TITLE
Added homework11

### PR DESCRIPTION
**Pull request by DaniilKorinnyi**
Homework11 was added

### CODE REVIEW

File: homework11/task2.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# ==== task 3 ====
+class MyStr:
+    def __init__(self, str):
+        self.str = str
+        print(str.upper())
+MyStr("hello")
+
+# ==== task 4 ====
+
+# class User:
```
Comment (klymenok) [inline homework11/task2.py:12]: remove commented code or uncomment it to be evaluated

File: homework11/task2.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# ======== additional tasks=======
+
+# ==== task 3 ====
+class MyStr:
+    def __init__(self, str):
+        self.str = str
+        print(str.upper())
+MyStr("hello")
```
Comment (klymenok) [inline homework11/task2.py:8]: Your class does't work like this:
```
my_str = MyStr('test')
print(my_str)
>>> "TEST"
```

To make it works you should override `__str__` method - print uses this method to get a string representation for objects.

File: homework11/task2.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# ==== task 4 ====
+
+# class User:
+#     def __init__(self, name):
+#         self.name = name
+#
+#     def __eq__(self, other):
+#         return self.name.lower() == other
```
Comment (klymenok) [inline homework11/task2.py:17]: Almost. Add `other.name.lower()` and it should work

**Review by klymenok — CHANGES_REQUESTED** [review]

**Review by DaniilKorinnyi — COMMENTED** [review]
Changed homework12

**Comment by DaniilKorinnyi** [comment]
> Looks good, but I guess homework 12 shouldn't be here

Yes, I didn't see that I'd committed to this branch. I reverted the commit, but nothing changed. Maybe the reason  is that I had already sent for reviewing. Or something else, don't you know?

**Pull request merged**